java-build/entrypoint.sh

```diff
@@ -50,13 +50,11 @@ else
   packageVersion="1.0.$productBuildNumber"
 fi
 
-if [ ! -z $productBranch ] && [ $productBranch != "master" ]; then
-  packageVersion="$packageVersion-$productBranch"
-fi
-
-echo "deploying version: '$packageVersion'"
+# deploy to Artifactory only on master
+if [ ! -z $productBranch ] && [ $productBranch = "master" ]; then
```

Comment:
We may be able to take this logic check out as well and move toward only calling this as a "CD" script from the calling repo on a merge to master change?

Reply:
This script also builds the thing which we want to test on each commit so we could do it but not without some extensive refactoring.